tools/js2c.py

```diff
@@ -288,6 +288,11 @@ def JS2C(source, target):
         split = split[1:]
       name = '/'.join(split)
 
+    # if its a gypi file we're going to want it as json
+    # later on anyway, so get it out of the way now
+    if name.endswith(".gypi"):
+      lines = re.sub(r'#.*?\n', '', lines)
```

Comment:
Does python maybe offer a package that converts a dict into json? That would be cleaner than replacing characters.

Reply:
https://github.com/nodejs/node/pull/19140#pullrequestreview-101029824

Reply:
ah. missed that context. thanks.